index.js

```diff
@@ -16,4 +16,45 @@ if (closeBtn) {
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/195599090-fa0bc14f-4f4e-4d34-b223-101ef33f99b2.png)

**[Optional]**

- [ ] You have added a link of your MIT.md license file in your readme file to make it descriptive but i guess you have forgot to add an license file :smile: Kindly add a license file in your repository using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) :+1: